_posts/blog/2017-07-20-upday_blogathon.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: post
+title: "upday Blogathon 2017"
+description: Blogathon, an approach to motivate engineers to write blog posts.
+modified:
+categories: blog
+author: robert_bordo
+excerpt: Doing good is fine, but talking about it is even better. How we motivated engineers to write blog posts.
+tags: [blogathon, hackathon, blog post, writing, blogging]
+image:
+date: 2017-07-19T01:00:00-04:00
+---
+
+<left>
+    <picture>
+        <a href="http://upday.github.io/jobs/"><img src="/images/blog/blogathon_2017/blogathon_2017_logo.png" 
+        width="10%" height="10%"
+        alt="Blogathon 2017 Logo" ></a>
+    </picture>
+</left>
+
+## What and why?
+
+Here at upday we have a lot of talented engineers doing very interesting “stuff” and using up-to-date (some would even say top-notch) technologies. 
+**Doing good is fine, but talking about it is even better.** The problem is that not every engineer likes writing about his profession. 
+Often I hear: *“this wouldn’t be interesting to anyone else”, “there are already articles about it”, “I don’t have time for this” or “I simply cannot write”.* 
+All this might be true. But for me, it’s not enough of an excuse.
+
+Last year we started our [upday tech blog](http://upday.github.io/) to attract more talented people and of course, 
+to get attention in the tech community. This worked out quite well. We received promising applications, 
+a few of us got invited as speakers to international conferences and we also organised well-received meetups. 
+Unfortunately, the driving force behind most of these activities left the company, hired by a much bigger player 
+(another proof our efforts bear fruit). Since then our blog has been more or less inactive.
```

Comment:
Here although the sentence grammatically is correct. It sounds without ending. I would add "and" just before the Since then[...]